### DISCUSSION TITLE
Got same result on different calls of VideoSequential

### DISCUSSION THREAD

**Discussion by Chzzi** [kornia/kornia#2276] [Questions]
I'm trying to generate two random augmented tensors from an original tensor. I declared a VideoSequential with components and used it twice, but I got the same results (output_1 and output_2). Do I have to declare two VideoSequential to get two different results?
```bash 
import kornia.augmentation as K
data = torch.ones(2, 3, 1, 5, 6).repeat(1, 1, 4, 1, 1) \
aug_list = K.VideoSequential(
        K.RandomHorizontalFlip(),
        K.ColorJitter(0.4, 0.4, 0.4, 0.1, p=0.8),
        K.ColorJiggle(0.1, 0.1, 0.1, 0.1, p=1.0),
        data_format='BCTHW',
        same_on_frame=True
    )
output_1 = aug_list(data)
output_2 = aug_list(data)
```

**Comment by edgarriba** [comment]
That definitely looks like a bug https://colab.research.google.com/drive/1bkBKQJfHJO7BG24idkC6BF1TbyHcfGLn

/cc @shijianjian @johnnv1 can you verify and open a tick in case is a bug

**Reply by Chzzi** [reply]
Does this mean that I need to redeclare the VideoSequential every time before I augment the tensor to ensure the randomness of the augmentation?

**Reply by shijianjian** [reply]
> Does this mean that I need to redeclare the VideoSequential every time before I augment the tensor to ensure the randomness of the augmentation?
> 
> 

You may use 'aug(image, params=aug.generate_parameters(image.shape))' as a temporary workaround.

**Reply by Chzzi** [reply]
> > Does this mean that I need to redeclare the VideoSequential every time before I augment the tensor to ensure the randomness of the augmentation?
> 
> You may use 'aug(image, params=aug.generate_parameters(image.shape))' as a temporary workaround.

Sorry, I don't know if I got that right. I used this code
`output_1 = aug_list(data, params=aug_list.generate_parameters(data.shape))` 
but got an error ‘VideoSequential object has no attribute generate_parameters’

**Reply by shijianjian** [reply]
> > > Does this mean that I need to redeclare the VideoSequential every time before I augment the tensor to ensure the randomness of the augmentation?
> 
> > 
> 
> > You may use 'aug(image, params=aug.generate_parameters(image.shape))' as a temporary workaround.
> 
> 
> 
> Sorry, I don't know if I got that right. I used this code
> 
> `output_1 = aug_list(data, params=aug_list.generate_parameters(data.shape))` 
> 
> but got an error ‘VideoSequential object has no attribute generate_parameters’

Sorry. It should be 'forward_parameters' instead.
